### PR TITLE
Added docs for soccer's graphing feature #144

### LINKED ISSUE #297: Document the robot status panel on the 'soccer' page in doxygen

### PR DESCRIPTION
**Pull request by imtn**
I've added an image of the screen on the soccer page. I don't have any descriptions, though, but can add them in if somebody can let me know what they are.

closes #297

### CODE REVIEW

File: clang [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/bin/clang
```
Comment (jgkamat) [inline clang]: I think this file can be removed (`git rm clang`, then a git commit)

**Comment by justbuchanan** [comment]
@imtn, I think the screenshot got added twice.  If you click on the "Files Changed" tab at the top of this pull request, you can see all changes made as part of this pull request.  I see two copies of the screenshot, so let's delete the one in the modeling folder.

**Comment by justbuchanan** [comment]
Also, just a minor thing, but it's best to only do work on branches other than master.  Try and keep the master branch of your repository in sync with the main RoboJackets one, then make a new "topic branch" for any work you do.

**Comment by imtn** [comment]
@jgkamat I've removed clang.
@justbuchanan I took out the second panel image. Also, thanks for the tip, I'll keep that in mind for next time.

**Comment by justbuchanan** [comment]
Looks good!  I just realized that I had this issue mixed up with a different one, so my mistake.  I changed the issue title and number to reference the graphing feature, rather than the status panel, since that's what you've added docs for here.

I'll go ahead and add in the description to the docs page and get it merged in.

**Comment by ashaw596** [comment]
Is the Docif submodule commit change suppose to be in this pull request?

**Comment by justbuchanan** [comment]
No, the DoCIF submodule shouldn't change in this PR.  I branched off of this [here](https://github.com/RoboJackets/robocup-software/tree/imtn/graphing-docs) and added in some more documentation text and reverted the submodule change.

**Pull request merged**